src/components/tooltip/Tooltip.css

```diff
@@ -0,0 +1,40 @@
+.stx-tooltip-wrapper {
+  display: inline-block;
+  position: relative;
+}
+
+.stx-tooltip {
+  position: absolute;
+  z-index: 9999999;
+  top: 100%;
+  margin-top: 1rem;
+  left: 50%;
+  transform: translateX(-50%);
+  padding: 10px;
+  border-radius: 6px;
+  background: rgba(0, 0, 0, 0.7);
+  color: white;
+  text-align: center;
+  display: none;
+  min-width: 120px;
+  max-width: 160px;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.stx-tooltip::before {
+  content: " ";
+  left: 50%;
+  bottom: 100%;
+  transform: translateX(-50%);
+  height: 0;
+  width: 0;
+  position: absolute;
+  pointer-events: none;
+  border: 8px solid transparent;
+  border-bottom-color: rgba(0, 0, 0, 0.7);
+}
+
+.stx-tooltip-wrapper:hover .stx-tooltip {
```

Comment:
question: how well does this work on a touch screen? I'm fairly sure the bevy tooltip appears on click.

Reply:
Just relying on Chrome's mobile view in the developer tools, a click on an element with the tooltip yields the same results as desktop

<img width="247" alt="image" src="https://github.com/bold-commerce/stacks-ui/assets/26397730/32052327-0fe6-431e-b1bf-dbce0d7b5c91">